test/Fetch.fs

```diff
@@ -105,93 +106,6 @@ let ``Post url encoded with return expression is Ok`` () =
         test <@ urldecoded'.Contains "foo=bar" @>
     }
 
-// [<Fact>]
```

Comment:
why this was commented out?

Reply:
Retry not supported anymore so we're using Polly instead.